README.md

```diff
@@ -3,12 +3,73 @@ o-card [![Circle CI](https://circleci.com/gh/Financial-Times/o-card/tree/master.
 
 Origami component for a card.
 
-## Types of Card
+## Browser support
```

Comment:
I don't think we should add this anymore as all our modules should follow the FT's browser support doc